### PR TITLE
fixed a typo and pass entrypoint to the saf cli to oclif

### PR DESCRIPTION
**Pull request by Amndeep7**
#44

### CODE REVIEW

File: run_command.js [hunk 1]
```diff
@@ -18,11 +19,11 @@ async function runCommand(overrideCommand) {
 
     const command = saf_command[0].includes(':') ? saf_command[0].split(':')[1] : saf_command[1];
 
-    if (topic == "view" & command == "heimdall") {
+    if (topic == "view" && command == "heimdall") {
         throw new Error("The SAF Action does not support the 'view heimdall' command. Please reference the documentation for other uses.");
     }
 
-    return saf.run(saf_command);
+    return saf.run(saf_command, resolve("./node_modules/@mitre/saf/lib/index.js"));
```
Comment (em-c-rod) [inline run_command.js:26]: What effect does it have to explicitly state the path to the saf cli run command?

Comment (em-c-rod) [reply run_command.js:26]: I see as stated in the issue, the SAF action was not resolving certain references correctly due to not knowing it was being called outside of the original context.

**Comment by em-c-rod** [comment]
Tested in another test repo to confirm the action is called correctly from this branch.

**Review by em-c-rod — APPROVED** [review]

File: run_command.js (hunk 1 above)
Comment (Amndeep7) [reply run_command.js:26]: Yup

**Pull request merged**